terraform/modules/jenkins/jenkins/snapshots.tf

```diff
@@ -0,0 +1,83 @@
+
+resource "aws_iam_role" "snapshot_jenkins_data_role" {
```

Comment:
We're hoping to move to all policy documents eventually. https://www.terraform.io/docs/providers/aws/d/iam_policy_document.html

Can we do these with policy docs?

Reply:
Ooh let's try it